project/build.sbt

```diff
@@ -11,6 +11,7 @@ Compile / unmanagedSourceDirectories ++= {
   ).map(dir => root / s"$dir/src/main/scala")
 }
 
+addSbtPlugin("de.heikoseeberger"  % "sbt-header"        % "5.0.0")
```

Comment:
It seems that you're not using `sbt-header` anywhere. At least I haven't seen its configuration, nor invocations of it in the CI scripts, etc.

I recommend removing it if it's not going to be used.